metadeploy/api/models.py

```diff
@@ -223,7 +231,23 @@ def valid_token_for(self):
         return None
 
 
-class ProductCategory(TranslatableModel):
+class Token(SiteRelated, BaseToken):
+    """
+    Identical to DRF's Token, but allows users to have one token per Site, instead of
+    one token in general.
+    """
+
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+
+    class Meta:
+        unique_together = ("user", "site")
+
+
+# Patch DRF token authentication to use our own model
+TokenAuthentication.model = Token
```

Comment:
This is a bit hacky but the test for tokens would catch if it breaks. The proper way is to create a subclass of `TokenAuthentication` just to set the `model` attribute and tell DRF to use that class. I can do that if preferred.